bin/opm

```diff
@@ -1509,6 +1510,19 @@ SKIP_INSTALL:
 }
 
 sub get_install_dir () {
+
+    if (defined $install_into_dir && defined $install_into_cwd) {
+        err "conflicting arguments: --cwd and --install-dir cannot both be used at the same time.\n";
```

Comment:
@SirR4T `err` is our own enhanced `die` so we should use `err` wherever possible.